source/en/api_bandit.rst

```diff
@@ -0,0 +1,145 @@
+Bandit
+------
+
+* See `IDL definition <https://github.com/jubatus/jubatus/blob/master/jubatus/server/server/bandit.idl>`_ for detailed specification.
+.. * see :doc:`method` for detailed description of algorithms used in this server.
+
+Configuration
+~~~~~~~~~~~~~
+
+Configuration is given as a JSON file.
+We show each field below:
+
+.. describe:: method
+
+   Specify bandit algorithm.
+   You can use algorithms below.
```

Comment:
I think "the algorithms below" is better.
A good example of "below" is the sentence from English tests such as TOEIC: "Read the following dialogue and answer the questions below".